.travis.yml

```diff
@@ -13,7 +13,7 @@ cache:
 install: true
 
 script:
-  - ./gradlew -s -Dorg.gradle.daemon=false clean build publishToMavenLocal
+  - ./gradlew -s -Dorg.gradle.daemon=false clean build -xcheckstyleMain -xfindBugsMain -xpmdMain war publishToMavenLocal
```

Comment:
At the very least we should leave checkstyle if only to get a consistent look and feel throughout the codebase. The others I would just comment out in the `gradle/projects.gradle` file.

Reply:
Checkstyle has about 60 errors remaining. Almost all of them are midding javadoc. I'll disable these as you suggested.